src/main/java/com/troubleshooting/service/ValidateService.java

```diff
@@ -0,0 +1,11 @@
+package com.troubleshooting.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class ValidateService {
+
+    public boolean validateInput(String containerId, String query) {
```

Comment:
If you the input might change in the future it's better to wrap it in a class (e.g. Input)